lang/en/tool_cloudmetrics.php

```diff
@@ -75,6 +75,8 @@
 $string['newusers_desc'] = 'Users who have signed up recently.';
 $string['dailyusers'] = 'Daily users';
 $string['dailyusers_desc'] = 'Unique users over a day (fixed frequency).';
+$string['rollingactiveusers'] = 'Rolling active users';
```

Comment:
Can you please name this 'Yearly active users'. I'm ok with this just being a language pack change but happy if you rename the classes and vars too to be consistent